### PR TITLE
Views

### PR DESCRIPTION
**Pull request by mp277**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -8,17 +8,23 @@ import Interests from './components/Interests';
 import Pending from './components/Pending';
 import Matches from './components/Matches';
 import Signup from './components/Signup';
-import Login from './components/Login';
+import Signin from './components/Signin';
```
Comment (aswils) [inline client/src/App.jsx]: this component changed to Login

File: client/src/App.jsx [hunk 2]
```diff
@@ -8,17 +8,23 @@ import Interests from './components/Interests';
...
+import Axios from 'axios';
 
 class App extends React.Component {
   constructor (props) {
     super(props);
     this.state = {
       username: "",
-      isLoggedIn: false,
+      isLoggedIn: true,
```
Comment (aswils) [inline client/src/App.jsx]: do we want to assume this?

File: client/src/App.jsx [hunk 3]
```diff
@@ -8,17 +8,23 @@ import Interests from './components/Interests';
...
     }
     this.transition = this.transition.bind(this);
   }
+  backHome() {
+    Axios.get('/', {
+
+    })
```
Comment (aswils) [inline client/src/App.jsx]: what's going on here? is this a stub?

File: client/src/App.jsx [hunk 4]
```diff
@@ -53,30 +59,19 @@ class App extends React.Component {
...
+        <Switch>
+          <Route path="/Profile" component={Profile} />
+          <Route path="/Interests" component={Interests} />
+          <Route path="/Pending" component={Pending} />
+          <Route path="/Matches" component={Matches} />
+        </Switch>
+      :
+        <Switch>
+          <Route path="/Signup" component={Signup} />
+          <Route path="/Signin" component={Signin} />
```
Comment (aswils) [inline client/src/App.jsx]: I think this is out of date

File: client/src/components/Pending.jsx [hunk 5]
```diff
@@ -1,15 +1,47 @@
...
+import Container from 'react-bootstrap/Container';
+import Tabs from 'react-bootstrap/Tabs';
+import Tab from 'react-bootstrap/Tab';
+import Toast from 'react-bootstrap/Toast';
+import ListGroup from 'react-bootstrap/ListGroup';
+import ListGroupItem from 'react-bootstrap/ListGroupItem';
 
 const Pending = () => {
-  
+  let users = [{ name: "Tiny", gender: "male", age: 21, img: "https://i.imgur.com/T7vjtfH.jpg" }, { name: "Tiny", gender: "male", age: 21, img: "https://i.imgur.com/9surkzy.jpg" }, { name: "Tiny", gender: "male", age: 21, img: "https://i.imgur.com/gnuDKcN.jpg"}];
```
Comment (aswils) [inline client/src/components/Pending.jsx]: let's keep fake data separate from the actual app. the app should work with users, not fake data

File: client/src/components/Profile.jsx [hunk 6]
```diff
@@ -5,19 +5,36 @@ import Card from 'react-bootstrap/Card';
...
+import ListGroup from 'react-bootstrap/ListGroup';
+import ListGroupItem from 'react-bootstrap/ListGroupItem';
+
 
 const Profile = (props) => {
 
   return (
+    <div>
+    <Card style={{ width: '24rem' }}>
+      <Card.Img variant="top" src="https://im.indiatimes.in/content/itimes/photo/2016/Oct/5/1475667154-funniest-baby-jokes-ever.jpg" />
```
Comment (aswils) [inline client/src/components/Profile.jsx]: same as above, let's keep fake data separate unless these are supposed to be defaults

File: client/src/App.jsx (hunk 3 above)
Comment (mp277) [reply client/src/App.jsx]: Yea its a function that should fire when the home button is clicked to go back to the index

File: client/src/components/Profile.jsx (hunk 6 above)
Comment (mp277) [reply client/src/components/Profile.jsx]: Gonna replace this image with a generic one for users who have not yet put a profile or otherwise for testing

**Pull request merged**